docs: add spectral ruleset docs (#790)
asyncapi/parser-js: docs/ruleset/README.md

# AsyncAPI Spectral Ruleset

> ParserJS has some built-in Spectral rulesets that validate AsyncAPI documents and inform about good practices. The distinctions are:

Reviewer: Simplifying

```suggestion
ParserJS has some built-in Spectral rulesets that validate AsyncAPI documents and inform about good practices. Those are:
```
